Documentation/devel/pod-lifecycle.md

```diff
@@ -83,22 +95,44 @@ All directories found in `$var/run` are tested for exited status by trying to ac
 
 When a directory's lock cannot be acquired, the directory is skipped as it indicates the pod is currently executing.
 
-For directories which the lock is successfully acquired, the directory is renamed from `$var/run/$uuid` to `$var/exited-garbage/$uuid`.  The renaming effectively implements the "mark" operation.  The locks are immediately released, and operations like `rkt status` may occur simultaneous to `rkt gc`.
+For directories which the lock is successfully acquired, the directory is renamed from `$var/run/$uuid` to `$var/exited-garbage/$uuid`.
```

Comment:
When the lock is successfully acquired, the directory is renamed from `/var/run/$uuid` to `/var/exited-garbage/$uuid`.